README.md

```diff
@@ -8,7 +8,11 @@ are available in the Amazon Elastic Container Registry (Amazon ECR).
 
 The AWS DLCs are used in Amazon SageMaker as the default vehicles for your SageMaker jobs such as training, inference, 
 transforms etc. They've been tested for machine
-learning workloads on Amazon EC2, Amazon ECS and Amazon EKS services as well.
+learning workloads on Amazon EC2, Amazon ECS and Amazon EKS services as well. 
+
+Starting with MXNet 1.9, PyTorch 1.10, and Tensorflow 2.7, CPU and GPU DLCs are released as "E3" and "SageMaker" images.  
+E3 images are supported on EC2, ECS, and EKS, whereas SageMaker images are supported on Amazon SageMaker and contain additional
+packages to support the SageMaker experience.
```

Comment:
I think it makes more sense to have this come after the pre-existing text, as a `Note:`.

Especially because we have text explaining this at a more relevant section on the available_images.md file as well.

Reply:
nit: "Starting with MXNet 1.9, PyTorch 1.10, and Tensorflow 2.7, CPU and GPU Deep Learning Containers are released as either E3 or SageMaker images. E3 images are supported on Amazon EC2, Amazon ECS, and Amazon EKS, whereas SageMaker images are supported on Amazon SageMaker and contain additional packages to support SageMaker workloads."

As is, "E3 and SageMaker" makes it sound like there are times when a DLC will be released as both. If this is not the case and there will always be separate releases, change to "either E3 or SageMaker images."

+ Changed "the SageMaker experience" to "SageMaker workloads"